spring-kafka/src/main/java/org/springframework/kafka/listener/KafkaMessageListenerContainer.java

```diff
@@ -372,8 +379,41 @@ public void onPartitionsAssigned(Collection<TopicPartition> partitions) {
 				consumer.assign(new ArrayList<>(this.definedPartitions.keySet()));
 			}
 			this.consumer = consumer;
-			this.listener = listener;
-			this.acknowledgingMessageListener = ackListener;
+			Object theListener = listener == null ? ackListener : listener;
```

Comment:
OK

STET: verb
1.
let it stand (used as an instruction on a printed proof to indicate that a correction or alteration should be ignored).

😄